example/sample1/federateddeployment-template.yaml

```diff
@@ -23,5 +23,4 @@ items:
           spec:
             containers:
             - image: nginx
-              imagePullPolicy: Always
```

Comment:
(No action required) Do we want to continue using nginx for the walkthrough?  I switched the test fixture to use busybox because it is much lighter weight (2mb vs 100mb).

cc: @font

Reply:
We should probably keep using `nginx` for deployments *only*, as we rely on it to verify `FederatedService` works by curling the web server using the minikube IP and `NodePort`.